src/api/api.ts

```diff
@@ -60,11 +60,14 @@ export const api = {
     password: string
   ): Promise<AccessToken> =>
     (
-      await instance.post<EmptyBody>("/api/user/signup/", {
-        name: name,
-        email: email,
-        password: password,
-      })
+      await instance.post<EmptyBody>(
+        "https://waffleoverflow.shop/api/user/signup/",
```

Comment:
전체 주소로 표시하면 개발중엔 CORS 오류가 납니다.
```suggestion
        "/api/user/signup/",
```